src/ocf_cache_priv.h

```diff
@@ -191,4 +191,6 @@ static inline uint64_t ocf_get_cache_occupancy(ocf_cache_t cache)
 
 int ocf_cache_set_name(ocf_cache_t cache, const char *src, size_t src_size);
 
+uint8_t any_core_rotational(ocf_cache_t cache);
```

Comment:
`ocf_cache_is_any_core_rotational()`
Let's try to keep API at least a bit object oriented.